README.md

```diff
@@ -49,12 +57,12 @@ These exchanges should support all markets that they have, and will be simple to
 
 **Note**
 
-As of now **Poloniex** exchange only has one open socket that sends back all data. The above syntax is not recommeneded unless you are only following one symbol; as it will open up multiple sockets that return all data, and filter out your selections.
+As of now, **Poloniex** exchange only has one open socket that sends back all data. The above syntax is not recommeneded unless you are only following one symbol, as it will open up multiple sockets that returns all data, and filter out your selections.
 __________
 
 ### echoExchange()
 
-A simple printout of all open ticker quotes.
+A simple printout of all opened ticker quotes.
```

Comment:
s/`opened`/`open`

Changing this doesn't add any clarity; "open ticker quotes" are ticker quotes that are open (referring to at the current time) not ticker quotes that were at some point in the past opened.